security/cmd/istio_ca/main.go

```diff
@@ -229,6 +242,19 @@ func runCA() {
 		fatalf("failed to create secret controller: %v", err)
 	}
 
+	if opts.genWebhookSecrets {
+		err = util.GenerateWebhookSecrets(cs.CoreV1(), ca, []util.WebhookIdentity{
```

Comment:
I think it's better to run this as a go routine. This function could take some time that delays the cert provision for K8s service accounts.